Jasmine/Grid/ViewModels/GridViewModel.swift

```diff
@@ -1,28 +1,43 @@
 import Foundation
 
 class GridViewModel: GridViewModelProtocol {
+    /// Stores the grid data that will be used to display in the view controller.
+    private(set) var gridData: [Coordinate: String] = [:] {
+        didSet {
+            delegate?.updateGridData()
+        }
+    }
+    /// Answers for this game. The game is won if this is done
+    private var answers: [[String]] = [["1", "2", "3", "4"], ["5", "6", "7", "8"],
+                                       ["9", "10", "11", "12"], ["13", "14", "15", "16"]]
     /// The delegate that the View Controller will conform to in some way, so that the Game Engine
     /// View Model can call.
     weak var delegate: GridGameViewControllerDelegate?
-    /// Stores the grid data that will be used to display in the view controller.
-    private(set) var gridData: [Coordinate: String] = [:]
     /// Specifies the current score of the game. If the game has not started, it will be the initial
     /// displayed score.
-    private(set) var currentScore: Int = 0
+    private(set) var currentScore: Int = 0 {
+        didSet {
+            delegate?.redisplay(newScore: currentScore)
+        }
+    }
     /// The timer of this game.
     private(set) var timer = CountDownTimer(totalTimeAllowed: Constants.Grid.time)
     /// The status of the current game.
     private(set) var gameStatus: GameStatus = .notStarted {
         didSet {
-            delegate?.notifyGameStatus()
+            delegate?.notifyGameStatusUpdated()
+
+            if gameStatus != .inProgress {
+                timer.stopTimer()
+            }
         }
     }
 
     /// Tells the view model that the game has started.
     func startGame() {
+        loadGrid(from: answers)
         timer = createTimer()
```

Comment:
why is there a createTimer, when the timer property has a pre-instantiated Timer object?